packages/nigeria/README.md

```diff
@@ -0,0 +1,68 @@
+# 🇳🇬 Nigeria
+
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-blue.svg)](https://creativecommons.org/licenses/by/4.0/)
+
+Contains all 36 states of the Federal Republic of Nigeria(including the FCT):
```

Comment:
To be consistent with other maps: `Contains all the states and the Federal Capital Territory (FCT) of Nigeria:`